docs/releases/handling-releases.md

```diff
@@ -146,27 +159,46 @@ _Outcome_: **Customers can install/update via WPORG; WPORG plugin page is up to
 #### Update `main` branch with release changes
 
 -   Merge the release branch back into `main` (without the branch being up to date with `main`). This may have merge conflicts needing resolved if there are cherry-picked commits in the release branch.
--   Do not delete the branch (release branches are kept open for potential patch releases for that version)
+-   Restore the branch if it is deleted. Release branches are kept open for potential patch releases for that version.
 -   For _major_ & _minor_ releases, update version on `main` with dev suffix, e.g. [`2.6-dev`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/commit/e27f053e7be0bf7c1d376f5bdb9d9999190ce158).
 
+_Outcome:_ __Main branch contains all changes and metadata tweaks for the release.__
+
+_Outcome:_ __Release branch is available for subsequent patch/bug fix releases.__
+
+_Outcome:_ __Dev version number on new builds of `main` branch.__
+
 #### Clean up release milestone / Zenhub
 
--   Edit the milestone and add the current date as the due date (this basically is used for easy reference of when the milestone was completed).
+-   Edit the (GitHub) milestone and add the current date as the due date (this basically is used for easy reference of when the milestone was completed).
 -   Close the milestone.
 -   If you didn't release a patch release, create a milestone for the next minor release.
 -   Close any epics that are completed as of this release and remove any unfinished issues in that from the epic.
 
+_Outcome:_ __The release milestone is archived and is an accurate record of what was included.__
+
+_Outcome:_ __There is a new release milestone ready for the next iteration.__
+
 #### Create pull request for updating the package in WooCommerce core.
 
-If the tagged release should be updated in WooCommerce core, do this immediately following our release.
+All releases (except RCs, betas etc) should be included in WooCommerce core. We do this by adding a PR on WooCommerce Core repo immediately after our release is completed.
```

Comment:
```suggestion
All releases (except pre-releases) should be included in WooCommerce core. We do this by adding a PR on WooCommerce Core repo immediately after our release is completed.
```

Reply:
Leaving this as is – `all releases except pre-releases` is less clear in my opinion.

Reply:
> is less clear in my opinion.

How so? Aren't RCs, betas pre-releases? The concern I have is the `etc` you tacked on there. What is `etc`?

Reply:
We can take off the etc - let's revisit this as we iterate.